resources/js/components/FileCard.vue

```diff
@@ -110,10 +121,26 @@ export default {
 
     methods:
     {
+        viewFile()
```

Comment:
No need for a new function you can pass the contentDisposition as a paramter to `downloadFile()` function and then to the `Nova.request().get()`